dali/pipeline/pipeline.h

```diff
@@ -438,6 +440,13 @@ class DLL_PUBLIC Pipeline {
 
   DLL_PUBLIC DISABLE_COPY_MOVE_ASSIGN(Pipeline);
 
+  /**
+   * Maps operator name (typically ExternalSource) to the
```

Comment:
can it be anything different than external source??

Reply:
Yes, in fact it will be `BatchSizeProvider` op. I'll update the comment